waiter/src/waiter/scaling.clj

```diff
@@ -372,6 +372,10 @@
                   (log/info "no target instances available for service"
                             {:scheduler-state scheduler-state, :service-id service-id})
                   {:scale-to-instances instances :target-instances target-instances :scale-amount 0}))]
+          (when (< instances (service-description "min-instances"))
+            (log/warn "Scheduler reported service had fewer instances than min-instances" {:service-id service-id
```

Comment:
Nitpick: I think we are trying to get all the log messages to start with lower-case

Reply:
Nitpick: Can we move the start of the map to the next line and have them all in a single line,

Reply:
done